StripePaymentsUI/StripePaymentsUI/Internal/UI/Views/STPFormTextField.swift

```diff
@@ -381,20 +400,18 @@ class STPTextFieldDelegateProxy: NSObject, UITextFieldDelegate {
         let hasTextContentType = textField.textContentType != nil
 
         if hasTextContentType && insertingIntoEmptyField && (string == " ") {
-            /* Observed behavior w/iOS 11.0 through 11.2.0 (latest):
```

Comment:
Looks like we lost this line

Reply:
I thought I fixed that, or maybe it happened in two places. Fixed.